ansible_collections/arista/avd/plugins/filter/convert_schema.py

```diff
@@ -8,37 +8,37 @@
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdtojsonschemaconverter import AvdToJsonSchemaConverter
 
 
-def convert_schema(schema: dict, type: str):
+def convert_schema(schema_id: str, type: str):
     """
     The `arista.avd.convert_schema` filter will convert AVD Schema to a chosen output format.
 
     Parameters
     ----------
-    schema : dict
-        Input AVD Schema
+    schema_id : str, ["eos_cli_config_gen" , "eos_designs"]
+        ID of AVD Schema
     type : str, ["documentation", "jsonschema"]
         Type of schema to convert to
 
     Returns
     -------
-    dict
+    dict | list
         Schema of the requested type
 
     Raises
     ------
     AvdSchemaError, AvdValidationError
         If the input schema is not valid, exceptions will be raised accordingly.
     """
-    avdschema = AvdSchema(schema)
+    avdschema = AvdSchema(schema_id=schema_id)
     if type == "documentation":
-        schemaconverter = AvdToDocumentationSchemaConverter(avdschema)
+        return AvdToDocumentationSchemaConverter(avdschema).convert_schema()
+
     elif type == "jsonschema":
-        schemaconverter = AvdToJsonSchemaConverter(avdschema)
+        return AvdToJsonSchemaConverter(avdschema).convert_schema()
+
```

Comment:
just noticed that the error message below is not really correct (not mentioning `jsonschema` type)

Reply:
fixed